### PR TITLE
order/customer integration

### PR DESCRIPTION
**Pull request by jao**

### CODE REVIEW

File: lib/abacos/customer.rb [hunk 1]
```diff
@@ -1,28 +1,51 @@
...
   class Customer
     @@mappings = {
       "email" => "EMail",
       "cpf_or_cnpj" => "CPFouCNPJ",
       "kind" => "TipoPessoa",
-      "gender" => "Sexo"
+      "gender" => "Sexo",
+      "birthday" => "DataNascimento",
+      "phone" => "Telefone",
+      "signupDate" => "DataCadastro"
```
Comment (jao) [inline lib/abacos/customer.rb]: .to_underscore

File: lib/abacos_integration/order.rb [hunk 2]
```diff
@@ -85,10 +93,13 @@ def customer_payload
...
         'email' => order_payload[:email],
         'cpf_or_cnpj' => order_payload[:cpf_or_cnpj],
-        'kind' => order_payload[:kind] || "tpeFisica",
-        'gender' => order_payload[:gender] || "tseFeminino",
-        'billing_address' => order_payload[:billing_address]
+        'kind' => order_payload[:billing_address][:kind] || "tpeFisica",
+        'gender' => order_payload[:billing_address][:gender] || "tseFeminino",
+        'birthday' => order_payload[:billing_address][:birthday],
+        'phone' => order_payload[:billing_address][:phone],
+        'signupDate' => order_payload[:billing_address][:signupDate],
```
Comment (jao) [inline lib/abacos_integration/order.rb]: to_underscore again.

File: lib/abacos_integration/order.rb [hunk 3]
```diff
@@ -85,10 +93,13 @@ def customer_payload
         'lastname' => order_payload[:billing_address][:lastname],
         'email' => order_payload[:email],
         'cpf_or_cnpj' => order_payload[:cpf_or_cnpj],
-        'kind' => order_payload[:kind] || "tpeFisica",
-        'gender' => order_payload[:gender] || "tseFeminino",
-        'billing_address' => order_payload[:billing_address]
+        'kind' => order_payload[:billing_address][:kind] || "tpeFisica",
+        'gender' => order_payload[:billing_address][:gender] || "tseFeminino",
```
Comment (jao) [inline lib/abacos_integration/order.rb]: why is gender and some other fields inside de billing_address ?

Comment (rovitulli) [inline lib/abacos_integration/order.rb]: for the same reason that firstname and lastname are in there originally too. Do you think we should move it?

**Comment by jao** [comment]
:+1:

**Pull request merged**